app/uk/gov/hmrc/plasticpackagingtax/registration/views/liability_process_more_weight_page.scala.html

```diff
@@ -0,0 +1,107 @@
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.sectionHeader
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.errorSummary
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.saveButtons
+@import uk.gov.hmrc.plasticpackagingtax.registration.controllers.{routes => pptRoutes}
+@import uk.gov.hmrc.plasticpackagingtax.registration.forms.ProcessMoreWeight
+@import uk.gov.hmrc.plasticpackagingtax.registration.forms.LiabilityLiableDate.yes
+@import uk.gov.hmrc.plasticpackagingtax.registration.forms.LiabilityLiableDate.no
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.paragraph
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.link
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.pageHeading
+
+@this(
+    formHelper: formWithCSRF,
+        pageHeading: pageHeading,
+        paragraph: paragraph,
+        link: link,
+    govukLayout: main_template,
+    govukRadios : GovukRadios,
+    sectionHeader: sectionHeader,
+    saveButtons: saveButtons,
+    errorSummary: errorSummary
+)
+
+
+@(form: Form[ProcessMoreWeight])(implicit request: Request[_], messages: Messages)
+
+
+@errorMessages = @{
+    val errors = (form.errors).map(err => messages(err.message, err.key)).mkString("<br>")
+    if(errors.nonEmpty) Some(ErrorMessage(content = HtmlContent(errors))) else None
+}
+
+@pptWeightQuestion = {
+@pageHeading(
+    text = messages("liabilityProcessMoreWeightPage.question")
+)
+}
+
+@pptGuidanceLink = {
+@link(
+    id = Some("guidance-link"),
+    target = "_blank",
+    text = messages("liabilityProcessMoreWeightPage.guidance.description"),
+    call = Call(
+        method = "GET",
+        url = messages("liabilityProcessMoreWeightPage.guidance.href")
+    )
+)
+}
+
+@pptWeightHelp = {
+@paragraph(
+    content = Html(messages("liabilityProcessMoreWeightPage.info"))
+)
+@paragraph(
+    content = Html(messages("liabilityProcessMoreWeightPage.info2", pptGuidanceLink))
+)
+}
+
+@govukLayout(
+    title = Title("liabilityProcessMoreWeightPage.title"),
+    backButton = Some(BackButton(messages("site.back"), pptRoutes.LiabilityWeightController.displayPage(), messages("site.back.hiddenText")))) {
+    @errorSummary(form.errors)
+
+    @sectionHeader(messages("liabilityProcessMoreWeightPage.sectionHeader"))
+
+    @pptWeightQuestion
+
+    @pptWeightHelp
+
+    @formHelper(action = pptRoutes.LiabilityProcessMoreWeightController.submit(), 'autoComplete -> "off") {
+
+    @govukRadios(Radios(
+        hint = None,
+        idPrefix = Some(form("answer").name),
+        name = form("answer").name,
+        items = Seq(
+            RadioItem(content = Text("Yes"), value = Some("yes"), checked = form("answer").value.contains(yes)),
+            RadioItem(content = Text("No"), value = Some("no"), checked = form("answer").value.contains(no))
```

Comment:
Can we hardcode "Yes" and "No" or should we read these values from messages. I think we'll need a Welsh version at some point?

Reply:
Yeah moved it messages file